src/styles/variables.less

```diff
@@ -18,6 +18,21 @@
 @size-button-borderRadius: @size-borderRadius;
 
 // colors
+@color-white: #fff;
+@color-black: black;
+
+@color-gray-5: fade(@color-black, 5%);
+@color-gray-10: fade(@color-black, 10%);
+@color-gray-25: fade(@color-black, 25%);
+@color-gray-30: fade(@color-black, 30%);
+
+// basic colors
+@color-ligthGray: #f4f4f4;
+@color-gray: #e3e3e3;
+@color-mediumGray: #c5c5c5;
+@color-darkGray: #333333;
```

Comment:
I agree we should consolidate these. Also, I'm assuming all those gray colors could be built using a `fade`.